src/libcollections/vec.rs

```diff
@@ -1165,6 +1165,15 @@ impl<T> Vec<T> {
         }
         other
     }
+    
+    pub fn remove_item<T: PartialEq>(&mut self, item: T) {
```

Comment:
I strongly recommend you try out `git gui`, especially in amend mode, and then force-pushing.